src/core/Entry.cpp

```diff
@@ -784,3 +791,29 @@ QString Entry::resolvePlaceholder(const QString& str) const
 
     return result;
 }
+
+QString Entry::resolveUrl(const QString& url) const
+{
+#ifdef WITH_XC_HTTP
+    QString newurl = url;
+    if (!url.contains("://")) {
+        // URL doesn't have a protocol, add https by default
+        newurl.prepend("https://");
```

Comment:
This will add `https` by default to every webUrl. The the webUrl is used for favicon downloading and for the KeePassHTTP protocol atm.
When downloading the favicon if the connection timeout or has an error, the protocol is downgraded to `http`.
You can test it with a website like `www.stealmylogin.com` that doesn't has `https` enabled